presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -41,27 +50,45 @@ Array Functions
 
     Concatenates the elements of the given array using the delimiter and an optional string to replace nulls.
 
+        SELECT array_join(ARRAY[1,1,2], '|'); -- 1|1|2
+        SELECT array_join(ARRAY[1,1,null], '|', 'NaN'); -- 1|1|NaN
```

Comment:
Use `'1|1|2'` as the output in comment, see examples here: https://prestodb.io/docs/current/functions/regexp.html

Let's have an example in which the array elements are varchar , and the delimiter is `,`